sql.txt

```diff
@@ -1,14 +1,38 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country 
+FROM Customers 
+LIMIT 10 
+OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * 
+FROM Customers 
+WHERE ContactName LIKE "A%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * 
+FROM OrderDetails 
+WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES (NULL, "Louisianna Sweet Heat", 2, 2, "12 - 20 oz bottles", 24),
+	   (NULL, "Sake", 1, 1, "50 - 1 liter bottles", 300),
+       (NULL, "Swordfish", 4, 8, "1 10 lb pkg", 75);
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE Products
+SET Price = Price + 1.5
+WHERE ProductID IN (SELECT TOP 2 ProductID FROM Products);
```

Comment:
Check please how it works again. This code gives me an error. It is enough to specify exact values you need, in your case it is ProductID 1 and 2. So, this part of the code will be  `WHERE ProductID IN (1, 2);`
There is also another solution
`UPDATE Products SET Price = (Price + 1.5) WHERE ProductID < 3;`